scripts/release-notes/release_notes.go

```diff
@@ -139,6 +139,13 @@ Download one of our static release bundles via our Google Cloud Bucket:
   - [cri-o.ppc64le.%s.tar.gz.spdx](https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.%s.tar.gz.spdx)
   - [cri-o.ppc64le.%s.tar.gz.spdx.sig](https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.%s.tar.gz.spdx.sig)
   - [cri-o.ppc64le.%s.tar.gz.spdx.cert](https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.%s.tar.gz.spdx.cert)
+- [cri-o.s390x.%s.tar.gz](https://storage.googleapis.com/cri-o/artifacts/cri-o.s390x.%s.tar.gz)
```

Comment:
Whites paces issue?

Reply:
Hm, in which sense?

Reply:
Ah, no. This is a new section. I check the whole file now. Looked like something was wrong with an indent.

Sorry for the commotion! Doh!